Formula/a/azion-cli.rb

```diff
@@ -1,6 +1,6 @@
 class AzionCli < Formula
```

Comment:
```suggestion
class Azion< Formula
```

Reply:
Do I rename the file as well or just the class?

Reply:
you need to change both, also remove bottle block, you can find examples in the github search